### PR TITLE
Extend TypeScript types

### PR DESCRIPTION
**Pull request by kibertoad**
refs #8

### CODE REVIEW

File: plugin.d.ts [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
 import { FastifyPluginAsync, FastifyReply, FastifyRequest } from 'fastify'
 
+declare module 'fastify' {
+  interface RouteShorthandOptions{
+    casbin?: {
+      rest?: boolean | {
+      getSub?: (request: FastifyRequest) => string,
+      getObj?: (request: FastifyRequest) => string,
+      getAct?: (request: FastifyRequest) => string
+    }
```
Comment (simoneb) [inline plugin.d.ts:12]: indentation seems slightly off here

Comment (kibertoad) [reply plugin.d.ts]: Fixed

**Review by simoneb — APPROVED** [review]

**Comment by kibertoad** [comment]
Sorry for hassle, but could you also publish this change? Would save us a bunch of castings :D

**Comment by simoneb** [comment]
Done in 1.1.1

**Comment by kibertoad** [comment]
Thank you!

**Comment by simoneb** [comment]
Thank you for the contributions!

**Comment by kibertoad** [comment]
It was my pleasure :)

**Pull request merged**